src/main/java/com/roommatematcher/enterprise/RoomMateController.java

```diff
@@ -26,4 +35,17 @@ public String index(Model model){
         model.addAttribute(profile);
         return "start";
     }
+
+    @PostMapping(value="/profile", consumes="application/json", produces="application/json")
+    public Profile createProfile(@RequestBody Profile profile) throws Exception {
+
+        Profile newProfile = null;
+        try{
+            newProfile = profileService.save(profile);
+        } catch (Exception e) {
+            //TODO ADD LOGGING
+            System.out.println("tried to access methods in business logic but failed");
+        }
```

Comment:
Is createProfile called anywhere?  I don't see any reference to it.

This looks like a new feature, with a TODO... which is increasing the technical debt of the codebase.